ui/src/App.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import logo from './logo.svg';
 import './App.css';
 import TextUpload from './TextUpload';
+import ImageUpload from './ImageUpload';
 
 const App = () => {
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <TextUpload />
+        <ImageUpload/>
```

Comment:
I think we need to eventually work on better styling than what we have, but that is for another time.